### ISSUE TITLE
2016-10-12 第六周周报

### ISSUE THREAD

**Issue by jetmuffin** [icsnju/X#6]
### 周报编写时间

每周三10：00 开会之前，编写周报时如果还没有创建，请按标题"YYYY-MM-DD 第X周周报"进行创建，并打上“周报”标签
### 周报格式

以姓名为标题,分“本周工作”，“下周工作"，“想说的话”三点进行阐述，内容自由发挥，切忌流水账，记录自己完成的工作，思考和分享，必要时给出具体的链接，重点内容加粗
### 周报示例

```
## xxx

#### 本周工作
- **xxx论文**  确定论文标题为"基于XXX的XXX技术的研究"，已完成论文的xxx部分，论文所在仓库地址为[xxx]()
- **xxx项目** 工作主要开始着手xxx项目，目前已经完成项目的xxx功能，主要采用xxx技术实现，实现时遇到xxx的问题，通过xxx方式解决，参考的文章链接为[解决xxx问题的方案](链接)
- ...

#### 下周工作
下周主要着手完成论文剩余部分和项目剩余功能
- **xxx论文**  阅读其他优秀论文，完成xxx章节
- **xxx项目**  下周项目组成员验收xxx功能，在周二之前需要完成xxx功能的开发，预计周末要多花时间
- ...

#### 想说的话
- 本周相对较忙，论文好赶啊，马上就要提交了
- 组会时间开太长了，需要提高效率，@曹总
- ...
```

**Comment by jetmuffin** [comment]
## 陈洁

#### 本周工作
- 部署了 docklet 运行环境，可以运行容器，对基础容器镜像进行了修改，可以通过容器对内外网进行访问。
- 完成了 docklet 集群的基础运维脚本编写（ansible 环境），建立运维的 playbook 管理仓库：[https://github.com/JetMuffin/cluster-ansible](https://github.com/JetMuffin/cluster-ansible)
- 与 @monkey-H 师兄讨论了 nap 的设计和改进想法，打算借鉴 kubernetes 进行改进。

#### 下周工作
- 发布、维护 docklet 平台，从中寻找问题进行并改进
- 尽快确定 nap 的修改思路，开始着手写代码
- 继续查找阅读关于 `usage rise/drop predict` 和 `resource oversubscription` 方面的论文

#### 想说的话
- 课好多，作业好多😅
- 好好干活，多向师兄们学习

**Comment by rdmclin2** [comment]
## 孟成林

#### 本周工作
- **Nata**  使用UML类图重新讨论了Nata的设计方案，完成稿见[nata.png](https://github.com/open-nata/nata-x/blob/master/nata.png)。更新Nata的[视觉稿](https://github.com/open-nata/nata-x/blob/master/Nata%E8%A7%86%E8%A7%89%E7%A8%BF.sketch),生成[pdf](https://github.com/open-nata/nata-x/blob/master/nata.pdf)，并发给中兴相关工作人员审查。
- **继续学习Angular2** 阅读并完成[英雄编辑器](https://angular.cn/docs/ts/latest/tutorial/toh-pt2.html),对比了Angular1和Angular2的运行机制。

#### 下周工作
- Push葛红军用Angular2.0完成第一版的首页,目前暂不考虑后台服务器,专注前端构建。
- 开始专注研究基于的Nata平台的迭代测试方法。

#### 想说的话
- 收心，干活

**Comment by luziling** [comment]
## 陆子凌

#### 本周工作
- 看Survey (Modeling, Profiling, and Debugging the Energy Consumption of Mobile Devices, CSUR'16)，没看完，看了第1~7章（22页）。
  
  大概内容是：先介绍统计移动设备总体能耗的方法；然后介绍建立移动设备能耗模型的方法；然后对现有的能耗统计工具进行分类，主要是按能耗模型的建立过程和能耗统计过程是on-device或off-device进行分类；最后介绍现有能耗统计工具的缺陷和面临的挑战。目前看到了分类中的第二类。

#### 下周工作
- 看完它。继续看论文。

#### 想说的话
- 好好干活。

**Comment by NAP-GHJ** [comment]
## 葛红军

#### 本周工作
- **Nata-git**  写了一个很小的nodejs的包，发布到npm服务器上了；现在对那个js-github已经能用它的接口，做到上层的git-commit和git-load文件，如果需要加接口应该也都可以；
- **学习Angular2** 阅读并完成[英雄编辑器](https://angular.cn/docs/ts/latest/tutorial/toh-pt2.html)，放到了github上[angular-quickstart](https://github.com/NAP-GHJ/angular-quickstart)
- **Nata-front** 在github刚建的一个[nata前端](https://github.com/NAP-GHJ/nata-front)的项目，刚刚起步

#### 下周工作
- 写代码，写界面
- 继续学各种js，angualr2框架，把以前看的东西逐渐用起来

#### 想说的话
- 干活
- 时间过得很快，进展来的很慢，提高效率！

**Comment by dmemoing** [comment]
## 邓靖

#### 本周工作
- 学习calico的使用方法，在两台主机上使用docker配置calico并相互ping通
- 阅读了docker-worker的代码，了解其使用ovs对网络的配置

#### 下周工作
- 对calico配置的网络性能进行测试，验证其性能是否更优
- 学习网络知识，增强对网络配置工具原理的理解

#### 想说的话
- 要学的很多，好好学习

**Comment by co2y** [comment]
## 王惟一

#### 本周工作
- 结合上周工作从字段的通用性和region两个角度寻求一种更通用的多列族存储模型。region方面没有找到优化方法；字段方面结合银行的交易流水表分析出在某些规则下多CF存储比之前的方法查询效率要高。

#### 下周工作
- 期望找到多CF存储的其它优势场景。
- 提取出多CF存储模型的意义。
- 补充实验验证猜想。

#### 想说的话
- 。。。

**Comment by qunnn41** [comment]
## 王逸群

#### 本周工作
- 确定目标，以Software Product Line Engineering和Feature Model为基础完成一个基于Constraint Solver的云平台配置管理工具。
- 利用EMF建立一个Cloud Platform的Metamodel，并用它产生不同的Feature Model.

#### 下周工作
- 利用Alloy Java API对Feature Model进行建模求解，对valid configuration能够进行有效验证。

#### 想说的话
- 加油

**Comment by monkey-H** [comment]
## 袁忠良

#### 本周工作
- 把maglev算法在nginx上实现了一部分，并没有全部完成，比想象中困难。
- 和余老师讨论了专利的想法，要把专利往云平台上靠拢，具体到云平台的应用场景。
-  @JetMuffin 讨论了nap，很佩服他的能力和想法。

#### 下周工作
- 继续完成maglev在nginx上的实现，并思考如何在上面做实验。
- 多和 @JetMuffin 交流，一起完善nap平台
-  继续和余老师交流。

#### 想说的话
- 继续加油

**Comment by ITanCh** [comment]
## 刘天驰

#### 本周工作
- 未从Input消息中发现有效地event生成方式，目前利用顾博的ape中的UIAutomation生成基本event。
- 已经将顾博的ape编译入系统，并且进行了一定的改造的定制。
- 设计和实现了新的测试工具，并且和monkey进行的简单的比较。

#### 下周工作
- 继续完善测试工具
- 多跑实验，收集数据，比较效果

#### 想说的话
- 。

**Comment by cshuo** [comment]
## 程硕

### 本周工作
- 尝试通过用Openstack N 版本的Nova替换原来的老版本, 尝试失败后, 阅读Nova及libvirt-python源码, 发现截至最新版本的libvirt-python都未对postcopy进行支持. 如要继续整合postcopy, 下一步可能设计到对libvirt-python代码的修改, 及重编译.
- 准备了讨论班报告.
- 搜集了部分AS自适应参数调整的论文 (未阅读).

### 下周工作
- 与ZC继续交流, 确定Timeline. 
- 阅读上周整理的论文, 总结可能的自适应调整方法, 确定简单的方案.  

### 想说的话
- 。。

**Comment by flyxu** [comment]
## 徐翔

#### 本周工作
- 了解了深度学习的一些常用模型，softmax, RBM, CNN,MLP等
- 了解了深度学习的一些常用框架，H2O, Tensorflow, Caffe, Theano等,公司让我先看H2O,另一个人在看Tensorflow,找他们之间的优缺点，做性能比较，以决定以后项目上使用哪个框架。
- 在实现之前提出的不平衡数据算法

#### 下周工作
- 看公司安排。
- 实现不平衡算法，然后用一些数据集做实验对比。

#### 想说的话
- 路上有点挤
